src/themes/createDarkTheme.ts

```diff
@@ -231,6 +231,27 @@ export function createDarkTheme(): Theme {
           },
         },
       },
+      CodeEditor: {
```

Comment:
Since the component is called WandelscriptEditor I would call it that here as well to match 🙂